src/models/recipients.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Will this create a separate collection? It should not create a sperate collection. The recipients should be embedded in the followup collection's documents.

Reply:
done